python/tk_desktop/desktop_window.py

```diff
@@ -56,10 +56,7 @@
 
 from .notifications import NotificationsManager, FirstLaunchNotification
 
-try:
-    from .extensions import osutils
-except Exception:
-    osutils = None
+from .extensions import osutils
```

Comment:
I've rewritten extensions so it looks like it always is there and we handle internally when it is missing. There's less of a chance of making a mistake by not testing for it now.